src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -47,6 +48,7 @@ public class EditCommand extends Command {
             + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
             + "[" + PREFIX_APPOINTMENT + "APPOINTMENT]"
+            + "[" + PREFIX_NOTE + "NOTE]"
```

Comment:
Based on user guide, it should follow NOTE -> APPOINTMENT -> TAG, but for now, its listed as APPOINTMENT -> NOTE -> TAG

Reply:
Good spot, thanks!